<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBookmarksWindow</class>
 <widget class="QDialog" name="EditBookmarksWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage bookmarks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ressources/ressources.qrc">
    <normaloff>:/img/Bookmark_32.png</normaloff>:/img/Bookmark_32.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="lwBookmarks">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbAddBookmark">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Add a new bookmark</string>
         </property>
         <property name="icon">
          <iconset resource="../../ressources/ressources.qrc">
           <normaloff>:/img/Plus-32.png</normaloff>:/img/Plus-32.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbDelBookmark">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete the selected bookmark</string>
         </property>
         <property name="icon">
          <iconset resource="../../ressources/ressources.qrc">
           <normaloff>:/img/Cross-32.png</normaloff>:/img/Cross-32.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbImport">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Import bookmarks</string>
         </property>
         <property name="icon">
          <iconset resource="../../ressources/ressources.qrc">
           <normaloff>:/img/Down_32.png</normaloff>:/img/Down_32.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbExport">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Export bookmarks</string>
         </property>
         <property name="icon">
          <iconset resource="../../ressources/ressources.qrc">
           <normaloff>:/img/Up-32.png</normaloff>:/img/Up-32.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox">
     <property name="title">
      <string>Edit bookmark</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Command :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="pteCommand"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Description :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="pteDescription"/>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ressources/ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditBookmarksWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditBookmarksWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
